### PR TITLE
LLM: add chatglm3 examples

### PR DESCRIPTION
**Pull request by JinBridger**
## Description

Add chatglm3-6b examples. Both for CPU and GPU.

### CODE REVIEW

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/chatglm3/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+## Example 1: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case for a ChatGLM3 model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations.
+### 1. Install
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.9
+conda activate llm
+
+pip install bigdl-llm[all] # install bigdl-llm with 'all' option
```
Comment (plusbang) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/chatglm3/README.md:16]: maybe `pip install --pre --upgrade bigdl-llm[all]`

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/chatglm3/README.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+## Example 2: Stream Chat using `stream_chat()` API
+In the example [streamchat.py](./streamchat.py), we show a basic use case for a ChatGLM3 model to stream chat, with BigDL-LLM INT4 optimizations.
+### 1. Install
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.9
+conda activate llm
+
+pip install bigdl-llm[all] # install bigdl-llm with 'all' option
```
Comment (plusbang) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/chatglm3/README.md:87]: similarly

File: python/llm/example/GPU/HF-Transformers-AutoModels/Model/chatglm3/streamchat.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    args = parser.parse_args()
+    model_path = args.repo_id_or_model_path
+    disable_stream = args.disable_stream
+
+    # Load model in 4 bit,
+    # which convert the relevant layers in the model into INT4 format
+    model = AutoModel.from_pretrained(model_path,
+                                      load_in_4bit=True,
+                                      trust_remote_code=True,
+                                      optimize_model=False)
```
Comment (plusbang) [inline python/llm/example/GPU/HF-Transformers-AutoModels/Model/chatglm3/streamchat.py:46]: In `python/llm/example/GPU/HF-Transformers-AutoModels/Model/chatglm3/generate.py`, parameter `optimize_model` is set to be `True`, why we need to set different parameter?

**Review by plusbang — COMMENTED** [review]
Please update link in these tables: https://github.com/intel-analytics/BigDL/blob/main/README.md#verified-models, https://github.com/intel-analytics/BigDL/blob/main/python/llm/README.md#verified-models, https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model#verified-models, https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/PyTorch-Models/Model#verified-models, https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU/HF-Transformers-AutoModels/Model#verified-models, https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU/PyTorch-Models/Model#verified-models

File: python/llm/example/CPU/HF-Transformers-AutoModels/Model/README.md [hunk 4]
```diff
@@ -10,6 +10,7 @@ You can use BigDL-LLM to run any Huggingface Transformer models with INT4 optimi
 | Falcon    | [link](falcon)    |
```
Comment (jason-dai) [inline python/llm/example/CPU/HF-Transformers-AutoModels/Model/README.md:10]: we can remove this table

File: python/llm/example/CPU/PyTorch-Models/Model/README.md [hunk 5]
```diff
@@ -6,6 +6,7 @@ You can use `optimize_model` API to accelerate general PyTorch models on Intel s
 |----------------|----------------------------------------------------------|
```
Comment (jason-dai) [inline python/llm/example/CPU/PyTorch-Models/Model/README.md:6]: we can remove this table

File: python/llm/example/GPU/HF-Transformers-AutoModels/Model/README.md [hunk 6]
```diff
@@ -9,6 +9,7 @@ You can use BigDL-LLM to run almost every Huggingface Transformer models with IN
 | Baichuan       | [link](baichuan)                                         |
```
Comment (jason-dai) [inline python/llm/example/GPU/HF-Transformers-AutoModels/Model/README.md:9]: we can remove this table

File: python/llm/example/GPU/PyTorch-Models/Model/README.md [hunk 7]
```diff
@@ -7,6 +7,7 @@ You can use `optimize_model` API to accelerate general PyTorch models on Intel G
 | Mistral        | [link](mistral)                                          |
```
Comment (jason-dai) [inline python/llm/example/GPU/PyTorch-Models/Model/README.md:7]: we can remove this table

**Review by plusbang — APPROVED** [review]
LGTM

**Pull request merged**